pkg/helper/client.go

```diff
@@ -260,6 +287,11 @@ func ApplyDesiredState(desiredState nmstatev1alpha1.State) (string, error) {
 		return pingOutput, rollback(fmt.Errorf("error pinging external address after network reconfiguration -> error: %v, currentState: %s", err, currentState))
 	}
 
+	err = checkApiServerConnectivity(defaultGwProbeTimeout * time.Second)
```

Comment:
We need to change the name of `defaultGwProbeTimeout` so it is shared for both and substract from it after we ping the GW. Or we can introduce another constant just for the apiserver.

Reply:
Adding new constant.